site/content/concepts/command-and-control.md

```diff
@@ -66,3 +66,31 @@ The response of the command will be sent by the device to
 If the device is authenticated, the `tenant` and `device-id` are left empty (resulting in 3 subsequent `/`s).
 
 ![Command & Control over MQTT Adapter](../command_control_concept_mqtt.png) 
+
+## Command & Control over AMQP Adapter
+
+When a device connected to the AMQP adapter wants to receive commands from the adapter, it opens a receiver link specifying the following source address:
+
+* `control` for authenticated devices
+* `control/${tenant}/${device-id}` for unauthenticated devices
+
+Once the receiver link is opened, the AMQP adapter sends command messages to devices through the link. The `subject` property of the message contains the actual command to be executed on the device while the `<request-id>`, used to correlate the command request with the response, is specified as an application property under key **APP_PROPERTY_REQUEST_ID**.
```

Comment:
We should either explicitly name the properties used to convey the information, e.g. 

> The *subject* property of the message contains the name of the command to be executed on the device while the *request_id* application property contains the identifier used to correlate the command request with the response.

or we should simply mention that the message contains the command's name and a request identifier in (unspecific) properties.